packages/shopify-codemod/README.md

```diff
@@ -12,6 +12,29 @@ This repository contains a collection of Codemods written with [JSCodeshift](htt
 
 ## Included Transforms
 
+### `global-identifer-to-import`
+
+Transforms global identifiers that you specify into the appropriate import statements. In order for this to work, you must set the `globalIdentifiers` option to an object where the keys are the names of globals used in your script, and the keys are the import paths for those globals.
```

Comment:
`Transforms global identifiers` seems misleading because this is an additive operation.

```diff
-the keys are the import paths
+the values are the import path
```

Overall, it could be more concise.  ```Creates import statements for global references.   Use the `globalIdentifiers` option to specify identifier/import path pairs.```